src/Misc/layoutbin/powershell/Get-LocalGroupMembership.ps1

```diff
@@ -0,0 +1,50 @@
+[CmdletBinding()]
+param()
+
+# Checks if a user is a member of a group using ADSI
```

Comment:
Could please describe in more detail how this test works?

Reply:
This is not a test :) I just used the [approved](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.2) verb for naming. I can rename it if needed

Reply:
@max-zaytsev I mean, could you please add some code comments to describe how the function works?